app/helpers/downloads_helper.rb

```diff
@@ -15,6 +15,11 @@ module DownloadsHelper
       title: 'File Geodatabase',
       commercialAvailable: true
     },
+    mpa_map: {
+      isDownload: false,
+      title: 'MPA Map',
+      url: URI.join(Rails.application.routes.url_helpers.root_url, 'MPA_Map.pdf')
```

Comment:
@stanleypliu Does this work for you? I have an error regarding the root_url which cannot be found.
However, I fell that's a bit unnecessary as the root url with the link tag can be achieve with just `/MPA_Map.pdf` which automatically fetches the file from the public folder.

Morover, I wonder if we should have a controller action that actually makes a download available for the user rather then opening the file up on the browser? I guess this is more a design question for @stacytalbot 
Otherwise we can just skip that for now to speed things up

Reply:
Have done, added a new key called `isMap` which I then also added in the `DownloadPopup` component and set the `download` HTML5 attribute to download the file named as it currently is.